BraintreeUIKit/Helpers/BTUIKAppearance.m

```diff
@@ -33,6 +33,8 @@ + (void)lightTheme {
     sharedTheme.activityIndicatorViewStyle = UIActivityIndicatorViewStyleGray;
     sharedTheme.useBlurs = YES;
     sharedTheme.postalCodeFormFieldKeyboardType = UIKeyboardTypeNumberPad;
+    sharedTheme.switchThumbTintColor = [UIColor whiteColor];
```

Comment:
The color constants are all class properties, so `UIColor.whiteColor`. Same applies to the `greenColor` below.

Reply:
The rest of the defaults are using the function call syntax [instead of the constants](https://github.com/braintree/braintree-ios-drop-in/blob/e8167e3fe078dbac45dbfb262888b705fc1fe050/BraintreeUIKit/Helpers/BTUIKAppearance.m#L19-L35). Shall we change them all for this PR?

Reply:
Sure, let's just modernize them all.

Reply:
Awesome. [Done](https://github.com/braintree/braintree-ios-drop-in/pull/145/commits/f110d9ef3b2ccdb8a5789e2bb2a45ab11ce194de) 👍